utils/docker/examples/daos-base/el8/Dockerfile

```diff
@@ -64,25 +64,23 @@ RUN	if [ -n "$BUST_CACHE" ] ; then
 	done &&                                                                                    \
 	echo "[INFO] Installing DAOS" ;                                                            \
 	dnf install                                                                                \
-		daos-${DAOS_VERSION}                                                               \
-		daos-client-${DAOS_VERSION}                                                        \
-		daos-client-tests-${DAOS_VERSION} &&                                               \
+		sudo                                                                               \
```

Comment:
I'm a little confused by this, is it installing sudo?  What is that package needed for?

Reply:
sudo is needed for some helpers scripts such as `run-daos_agent`,  `run-daos_server`, `daos-bash`